openbb_platform/core/openbb_core/app/static/container.py

```diff
@@ -37,7 +57,7 @@ def _check_credentials(self, provider: str) -> Optional[bool]:
         return all(getattr(credentials, r, None) for r in required)
 
     def _get_provider(
-        self, choice: Optional[str], command: str, default_priority: Tuple[str, ...]
```

Comment:
I had the idea that this (not using typing's type hints) would break on 3.9 envs, but it didn't, probably a 3.8 thing 😀